cryptofeed/binance/binance.py

```diff
@@ -126,21 +126,24 @@ def __reset(self):
                 ASK: sd({Decimal(ask[0]): Decimal(ask[1]) for ask in msg['asks']})
             }
 
-        await self.callbacks[L2_BOOK](feed=self.id, pair=self.pairs, book=self.l2_book, timestamp=time.time() * 1000)
+        await self.callbacks[L2_BOOK](feed=self.id, pair=pair, book=self.l2_book, timestamp=time.time() * 1000)
 
     async def message_handler(self, msg):
         msg = json.loads(msg, parse_float=Decimal)
 
+        # Combined stream events are wrapped as follows: {"stream":"<streamName>","data":<rawPayload>}
+        pair, event = msg['stream'].split('@')
         msg = msg['data']
 
-        if 'e' not in msg:
-            await self._book(msg)
+        # All symbols for streams are lowercase
+        pair = pair_exchange_to_std(pair.upper())
+
+        if event == 'depth20':
+            await self._book(msg, pair)
         elif msg['e'] == 'trade':
             await self._trade(msg)
         elif msg['e'] == '24hrTicker':
             await self._ticker(msg)
-        else:
-            LOG.warning("%s: Unexpected message received: %s", self.id, msg)
```

Comment:
why remove the log warning? Does the change to streams mean we'll get unhandled messages?